### ISSUE TITLE
テーマチェックのバグ修正

### ISSUE THREAD

**Issue by featherplain** [featherplain/amethyst#98]
- [x] REQUIRED: `.sticky` css class is needed in your theme css.
- [x] REQUIRED: `.bypostauthor` css class is needed in your theme css.
- [x] WARNING: Found wrong tag, remove `mobile-first` from your style.css header.
- [x] WARNING: Found a translation function that is missing a text-domain. Function esc_attr__, with the arguments 'Search'
- [x] WARNING: Found a translation function that is missing a text-domain. Function __, with the arguments 'Search for:'
- [x] WARNING: .ds_store Hidden Files or Folders found.

```
- INFO: Only one text-domain is being used in this theme. Make sure it matches the theme's slug correctly so that the theme will be compatible with WordPress.org language packs.
- The domain found is `amethyst`
```

**Comment by featherplain** [comment]
- [x] WARNING: Found wrong tag, remove single-column from your style.css header.

**Comment by featherplain** [comment]
merged at #99  and #100
